pkg/trace/api/api.go

```diff
@@ -355,10 +357,14 @@ func decodeTraces(v Version, req *http.Request) (pb.Traces, error) {
 		}
 		return tracesFromSpans(spans), nil
 	case v05:
+		buf := getBuffer()
+		defer putBuffer(buf)
+		_, err := io.Copy(buf, req.Body)
+		if err != nil {
```

Comment:
```suggestion
		if _, err := io.Copy(buf, req.Body); err != nil {
```
You could do this, but I don't feel strongly about it.